Gododir/gen.go

```diff
@@ -404,14 +414,16 @@ func generate(service *service) {
 	fmt.Printf("Generating %s...\n\n", service.Fullname)
 	delete(service)
 
-	autorest := exec.Command(fmt.Sprintf("%s/autorest/src/core/AutoRest/bin/Debug/net451/win7-x64/autorest", autorestDir),
+	autorest := exec.Command("gulp",
```

Comment:
Given that this is pretty new stuff, and some of our computers (okay, well mine) haven't had gulp setup yet. We may want to add in some guard rails like this:
[exec.LookPath(string)](https://godoc.org/os/exec#LookPath)